src/imgui/ImGuiMachine.cc

```diff
@@ -31,6 +31,36 @@ using namespace std::literals;
 
 namespace openmsx {
 
+void ImGuiMachine::save(ImGuiTextBuffer& buf)
+{
+	savePersistent(buf, *this, persistentElements);
+
+	for (const auto& item : recentMachines) {
+		if (!item.empty()) {
```

Comment:
I took it over from the loop in `ImGuiMedia.cc`, but indeed: I don't understand how it can be empty there, let alone here.... how can it be empty there?

Reply:
Should be fixed in the latest version.